default/main.py

```diff
@@ -70,6 +70,7 @@ async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
         for user in models.Userdata.query():
             async with aiohttp.ClientSession() as httpSession:
 
+                # run VT query
```

Comment:
I don't know any Google guide for that, but I'm a big fan of writing comments using sentences.
So in this case it should be:
```suggestion
                # Run VT query.
```